### PR TITLE
Update and warn the users on how to remove detailed geometries

### PR DESCRIPTION
**Pull request by ponceta**
Fix https://github.com/QGEP/QGEP/issues/590

### CODE REVIEW

File: en/user-guide/digitizing/deletingobjects.rst [hunk 1]
```diff
@@ -69,3 +69,13 @@ Delete Structure parts
 * Click the red x button to delete the structure parts
 * Click save on the form
 * Stop the editing mode and confirm changes to layer. All changes will be saved to database.
+
+Delete detailled geometries
```
Comment (m-kuhn) [inline en/user-guide/digitizing/deletingobjects.rst:73]:
```suggestion
Delete detailed geometries
```

File: en/user-guide/digitizing/deletingobjects.rst [hunk 2]
```diff
@@ -69,3 +69,13 @@ Delete Structure parts
...
+----------------------
+
+.. attention:: Removing features directly from **wastewater_structure** will remove these entities from the database and not only the detailled geometry!
```
Comment (m-kuhn) [inline en/user-guide/digitizing/deletingobjects.rst:76]:
```suggestion
.. attention:: Deleting features directly from **wastewater_structure** will remove the complete wastewater structure from the database and not only the geometry of the wastewater structure!
```

**Comment by ponceta** [comment]
Thanks @m-kuhn for the review!

**Review by m-kuhn — APPROVED** [review]

**Comment by m-kuhn** [comment]
merci

**Pull request merged**